src/meta/src/manager/catalog/mod.rs

```diff
@@ -380,6 +400,8 @@ where
         views.insert(view.id, view.clone());
         commit_meta!(self, views)?;
 
+        user_core.increase_ref_count(view.owner, 1);
```

Comment:
And please help to move updating `dependent_relation_id` after `commit_meta` to keep consistency.